mp/src/game/client/momentum/ui/HUD/hud_keypress.cpp

```diff
@@ -324,11 +346,19 @@ void CHudKeyPressDisplay::DrawKeyTemplates()
     // jump
     // Bullrush is Bhop being disabled
     surface()->DrawSetTextColor((m_nDisabledButtons & IN_JUMP || m_nDisabledButtons & IN_BHOPDISABLED) ? m_Disabled
-                                                                                                   : m_darkGray);
+                                                                                                       : m_darkGray);
     surface()->DrawSetTextPos(GetTextCenter(m_hWordTextFont, m_pwJump), jump_row_ypos);
     surface()->DrawPrintText(m_pwJump, wcslen(m_pwJump));
     // duck
     CHECK_INPUT_N(IN_DUCK);
     surface()->DrawSetTextPos(GetTextCenter(m_hWordTextFont, m_pwDuck), duck_row_ypos);
     surface()->DrawPrintText(m_pwDuck, wcslen(m_pwDuck));
+    CHECK_INPUT_N(IN_ATTACK);
+    int text_m1 = GetTextCenter(m_hTextFont, m_pwM1) - (UTIL_ComputeStringWidth(m_hTextFont, m_pwM1)) * 3.5;
+    surface()->DrawSetTextPos(text_m1, top_row_ypos);
+    surface()->DrawPrintText(m_pwM1, wcslen(m_pwM1));
+    CHECK_INPUT_N(IN_ATTACK2);
+    int text_m2 = GetTextCenter(m_hTextFont, m_pwM2) + (UTIL_ComputeStringWidth(m_hTextFont, m_pwM2)) * 2.75;
```

Comment:
I'm starting to wonder if there's a refactor opportunity here to store keys (including their templates) in their own class and do all this calculation inside of them...

The reason I bring this up is I'm seeing very similar positioning code being calculated here as in Paint, and not just for your added keys.